cmd/instrumentless_test/main.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
```

Comment:
can't this be a small bash script or a curl command we provide in maintainer docs?

Reply:
It is nice to have some way to test `GetCoupon` manually. Not sure what the better way is though.